Rakefile

```diff
@@ -3,6 +3,7 @@ require 'html-proofer'
 task :test do
   sh "bundle exec jekyll build"
   options = { :check_html => true, # :validation => { :report_missing_names, :report_invalid_tags => true, :report_script_embeds => true },
+              :cache => { :timeframe => '2d'},
```

Comment:
2 days is a random compromise between having a cache for a couple of PRs and also noticing not too long after when some links go dead (we could make do a 1 week cache if we did a nightly check or some such)